core-data/src/test/java/com/google/samples/apps/nowinandroid/core/data/repository/OfflineFirstUserDataRepositoryTest.kt

```diff
@@ -47,7 +47,7 @@ class OfflineFirstUserDataRepositoryTest {
     }
 
     @Test
-    fun offlineFirstTopicsRepository_toggle_followed_topics_logic_delegates_to_nia_preferences() =
+    fun offlineFirstUserRepository_toggle_followed_topics_logic_delegates_to_nia_preferences() =
```

Comment:
Nit: Technically the full name here is `OfflineFirstUserDataRepository` 😄

Reply:
Lol ty!